libraries/cmake/source/librdkafka/CMakeLists.txt

```diff
@@ -3,6 +3,13 @@
 #
 # This source code is licensed in accordance with the terms specified in
 # the LICENSE file found in the root directory of this source tree.
+# 
+#   flappy-sh added code blocks 
+#   openssl ,zstd, zlib dont get piked up correctly
+#   https://github.com/edenhill/librdkafka/blob/bd74dc31ba6edf431749b4ad5a00aa4fc78882fc/packaging/cmake/README.md
+#   'The cmake build mode is experimental and not officially supported, the community is asked to maintain and support this mode through PRs.'
+#   from the librdkafka win32 cmake , its ugly but it works
+# 
```

Comment:
Are these necessarily needed? I appreciate the note, but I think we tend to prefer keeping comments pretty minimal. Perhaps this could be changed to just be:
```
The cmake build mode is experimental and not officially supported
```
or something along those lines?

Reply:
coment updated